extensions/wikia/AdEngine/js/spec/video/player/porvata/googleImaSetup.spec.js

```diff
@@ -65,12 +65,13 @@ describe('ext.wikia.adEngine.video.player.porvata.googleImaSetup', function () {
 			request = imaSetup.createRequest({
 				width: slotWidth,
 				height: slotHeight,
+				useMegaAdUnitBuilder: true,
```

Comment:
We shouldn't change behaviour of all IMA instances. It should affect only ABCD ad product